cloudify-manager-worker/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+All notable changes start from v0.4.0 of this helm chart will be documented in this file.
+
+Provides a release information for each version splitted by the next sections
+
+- **Added** - for new added features
+- **Fixed** - for fixed issues
+- **Changed** - for changes in already implemented
+- **Removed** - for now removed features
+
+</br>
+
+# [v0.4.0] - 2022-11-08
+## Changed
+
+**BREAKING CHANGES:**
```

Comment:
Indeed important to specify that.
Just need to ensure that according to our conclusion with the .deploy false/true by default to update that log

Reply:
Sure, added to CHANGELOG and README